pkg/platform/kube/ingress/ingress.go

```diff
@@ -57,31 +56,39 @@ func (m *Manager) GenerateResources(ctx context.Context,
 	spec Spec) (*Resources, error) {
```

Comment:
why not adding the ingress PathType to `Spec` and read it from there? if empty, enrich with `networkingv1.PathTypeImplementationSpecific`

Reply:
This ingress spec is defined in the caller function from the apiGateway spec.
Do we want to add it to the apiGateway spec as well?

Or maybe leave it to a different PR?

Reply:
Add on this pr, same as you did with function config.